@syahrul-aiman 
There are some linting issues, please fix it:
plugins/modules/lvm_facts.py:310:23: E111 indentation is not a multiple of 4
plugins/modules/lvm_facts.py:311:23: E111 indentation is not a multiple of 4

--- a/plugins/modules/lvm_facts.py
+++ b/plugins/modules/lvm_facts.py
@@ -307,8 +307,8 @@ def load_pvs(module, name, LVM):
                 )
                 if stderr.find('0516-320') > -1:
                     if fields[2] == "None":
-                      del LVM['PVs'][pv]["VOLUME GROUP"]
-                      del LVM['PVs'][pv]["vg"]
+                        del LVM['PVs'][pv]["VOLUME GROUP"]
+                        del LVM['PVs'][pv]["vg"]
                 elif stderr.find('0516-010') > -1:
                     try:
                         LVM['PVs'][pv] = parse_pvs(module, stdout, pv)

